src/components/application_manager/include/application_manager/commands/mobile/alert_request.h

```diff
@@ -72,13 +72,6 @@ class AlertRequest : public CommandRequestImpl {
    **/
   virtual void Run();
 
-  /*
-   * @brief Will caled by request controller, when default will be expired.
-   * If Alert request has soft buttons, timeout response should not be sent to
-   * mobile
-   */
-  virtual void onTimeOut();
```

Comment:
Deprecate, don't remove

Reply:
@jacobkeeler all deleted signatures returned back and marked as deprecated. 
Body of this method was removed and called parrent:onTimeOut